submissions/morenko/task-frogger/js/app.js

```diff
@@ -0,0 +1,177 @@
+const countRows = 6;
+const countColumns = 5;
+
+const tileHight = 83;
+const tileWidth = 105;
+
+class Position {
+    startX;
```

Comment:
You don't need to initiate those variables here, they will be created from constructor.